### DISCUSSION TITLE
빌드 및 배포 작업 중 에러

### DISCUSSION THREAD

**Discussion by borimong** [devrel-kr/HackaLearn#64] [2 Q&A - 클라우드 스킬 챌린지] [answered]
![image](https://user-images.githubusercontent.com/86764406/128462164-5f2e9b08-5844-4b65-8b2e-6d75d727ae62.png)

react-app 을 빌드했음에도 자꾸 이와 같은 에러가 생겨서 질문드립니다..!! 어떻게 해결할 수 있을까요..?

**Comment by minhoryang** [comment, marked as answer]
강의에 해당한 템플릿을 통해 만든 리포가 아닌것으로 보입니다.

**Comment by justinyoo** [comment]
지금 스샷에 올라온 앱은 [Web Dev for Beginners](https://github.com/microsoft/Web-Dev-For-Beginners) 쪽을 사용중입니다. 우리 챌린지와는 상관 없는 부분이라 해결책을 드리기 어렵습니다.
